Base/usr/share/man/man7/kernel_cmdline.md

```diff
@@ -0,0 +1,129 @@
+## Name
+
+Kernel cmdline - boot arguments to control system functionality
+
+## Description
+
+The kernel supports boot parameters to change some of its behavior. This
+functionality can be used to disable or enable features in the OS entirely, but
+can sometimes be used also to change a feature's behavior without disabling it
+entirely.
+
+Some of the boot parameters require a value after it. In that case, you will
+need to write them like this:
+
+```
+param1=value ...
+
+```
+For those that don't require, you should just write them without equals sign,
+like that:
+
+```
+param1 ...
+
+```
+
+## Where to put those parameters
+
+You can either add kernel boot parameters when you invoke `./run`:
+
+```
+./run qcmd param1 param2 ...
+
+```
+for Q35 machines, use this layout:
+
+```
+./run q35_cmd param1 param2 ...
+
+```
+
+You may wish to write the kernel boot parameters in a more consistent method.
+For that, you can modify the GRUB configuration files. Since we have 3 of them
+(one for each partition scheme), you should pick the right one for you, so if
+you build a virtual disk with a GPT partition scheme, you should edit
+`grub-gpt.cfg` file.
+
+## Kernel boot parameters list
+
+Since it's pretty much not convenient for beginners to have a travel in the
+`init.cpp` file, the list here is meant to aid users to find all the kernel boot
+parameters and their meaning.
+
+### `text_debug`
+
+This boot parameter requires no value after it. This boot parameter enables
+Shell mode during boot, therefore you won't boot into the graphical mode.
+
+### `no_vmmouse`
+
+This boot parameter requires no value after it. This boot parameter disables the
+VMMouse feature. This boot parameter is useless on real hardware or emulators
+that don't support the VMMouse feature.
```

Comment:
Is there an explanation somewhere of what that feature is that we could link from here? If not another man page, maybe a web page...

And this should say that this makes *the PS2 mouse driver* always emit relative packets.

Reply:
I can write another document about VMWare backdoor if that's something we can use later.